packages/devtools_app_shared/CHANGELOG.md

```diff
@@ -3,15 +3,17 @@ Copyright 2025 The Flutter Authors
 Use of this source code is governed by a BSD-style license that can be
 found in the LICENSE file or at https://developers.google.com/open-source/licenses/bsd.
 -->
+## 0.3.1
+* Add "Supported Flutter versions" policy to the README.
```

Comment:
was this one supposed to move up to 0.3.1?

Reply:
I thought it wasn't included in the release notes for 0.3.0 so I included it here. I must have been looking at the wrong package on Pub.